Code.gs

```diff
@@ -12,12 +12,15 @@
 // var targetCalendarName = "" // The name of the Google Calendar you want to add events to
 // var sourceCalendarURL = "" // The ics/ical url that you want to get events from
 
-var sourceCalendars={}
+
+var sourceCalendars={
+  CustomName:"ICS/ICAL URL"
+}
```

Comment:
Rather than having `sourceCalendars` "prepopulated" with example data, let's leave it blank (like you did before with `var sourceCalendars = { };` ) and put a comment before explaining how it should be used (even use a block comment if you need to)